h2o-hive/src/main/java/water/hive/JdbcHiveMetadata.java

```diff
@@ -217,21 +222,24 @@ private StorableMetadata readPartitionMetadata(
         List<Column> partitionKeys,
         List<String> values
     ) throws SQLException {
-        String query = String.format(SQL_DESCRIBE_PARTITION, tableName, toPartitionIdentifier(partitionKeys, values));
-        Map<String, Object> data = executeAndParseJsonResultSet(conn, query);
+        String query = getDescribePartitionQuery(partitionKeys, values);
+        Map<String, Object> data = executeAndParseJsonResultSet(conn, query, tableName);
         Map<String, Object> info = (Map<String, Object>) data.get("partitionInfo");
         return readStorableMetadata(info);
     }
-
-    private String toPartitionIdentifier(List<Column> partitionKeys, List<String> values) {
+    
+    private String getDescribePartitionQuery(List<Column> partitionKeys, List<String> values) {
         StringBuilder sb = new StringBuilder();
+        sb.append(SQL_DESCRIBE_PARTITION).append("(");
```

Comment:
I might be in the minority here, but I actually think it is beneficial to the readability of what is going in the code to avoid these one-time constants and just use the strings directly as a caveman would.